src/core/agent/services/identifierService.ts

```diff
@@ -411,6 +417,10 @@ class IdentifierService extends AgentService {
       isPending,
       multisigManageAid: ourIdentifier,
     });
+    if (ourMetadata.groupMetadata) {
```

Comment:
This always has to be set

Reply:
same with https://github.com/cardano-foundation/cf-identity-wallet/pull/376#discussion_r1529952026